exercises/yacht/canonical-data.json

```diff
@@ -0,0 +1,261 @@
+{
+    "exercise" : "yacht",
+    "version" : "1.0.0",
+    "comments" : [
+        "The dice are represented always as a list of exactly five integers",
+        "with values beteen 1 and 6 inclusive. The category is an integer.",
+        "the numbers 0 to 5 represent the categories 'ones' to 'sixes'",
+        "Then 6 is full house,",
+        "     7 is four of a kind",
+        "     8 is a little straight 1-5",
+        "     9 is b big straight 2-6",
+        "    10 is Choise, sometimes called Chance",
```

Comment:
I don't think Choise is a correct spelling. I think it should be Choice, unless there is a game-specific spelling I am not aware of (in that case, simply make me aware of it)